Microsoft.NET.Build.Containers/Image.cs

```diff
@@ -18,8 +18,12 @@ public class Image
 
     private readonly Dictionary<string, string> _environmentVariables;
 
-    public Image(ManifestV2 manifest, JsonNode config)
+    public bool IsWindows => RuntimeIdentifier.StartsWith("win", StringComparison.OrdinalIgnoreCase);
+    public string RuntimeIdentifier { get; }
+
+    public Image(ManifestV2 manifest, JsonNode config, string runtimeIdentifier)
     {
+        RuntimeIdentifier = runtimeIdentifier;
```

Comment:
We should not need this - the `config` JsonNode has the `os` and `architecture` properties, which we should be able to use to answer this question appropriately.

Reply:
Changed it to use the `os` of the base image config.